airbyte-integrations/connectors/source-intercom/setup.py

```diff
@@ -6,23 +6,23 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk==0.29.0",
+    "airbyte-cdk~=0.40",
 ]
 
 TEST_REQUIREMENTS = [
     "pytest",
-    "pytest-mock",
+    "requests-mock",
```

Comment:
this is the dependency we used before [the migration to low-code](https://github.com/airbytehq/airbyte/commit/78828413a222f55fbd851a85accfb19124d95005#diff-b8515d841d0010d0b536b78ba5354491f42115d0982e7f95eb2cff299c8d10abR14)